env.sample

```diff
@@ -0,0 +1,5 @@
+COMPOSE_FILE=
+
```

Comment:
are we using this?

Reply:
yes, we are using this it is used for the [docker compose merge of frontend and backend](https://docs.docker.com/compose/how-tos/environment-variables/envvars/#compose_file)